Lesson8/Task3.py

```diff
@@ -0,0 +1,12 @@
+def make_operation(operator, *args):
+    if operator == '+':
+        return sum(args)
+    else:
+        print('Invalid operator')
```

Comment:
According to the task your function should work not only with `+` operator, but at least with `*` and `-`.
Please add additional logic

Reply:
Added more operators to function, please review @Masergio .